### DISCUSSION TITLE
Create `.github/release.yml` file?

### DISCUSSION THREAD

**Discussion by spenserblack** [o2sh/onefetch#1175] [Ideas]
I'm not sure how GitHub releases are currently being made. Are the changes being randomly categorized? Just an FYI that you can create a `.github/release.yml` file and use labels to categorize PR titles into appropriate sections.

See https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#configuring-automatically-generated-release-notes

**Comment by o2sh** [comment]
Done https://github.com/o2sh/onefetch/pull/1177. 

Hopefully I won't forget to tag the pull request with the correct labels.
BTW, I also removed all the emojis from the labels to simplify the config files.
